pkg/test/expectations/expectations.go

```diff
@@ -699,5 +700,18 @@ func EventuallyExpectTerminating(ctx context.Context, c client.Client, objs ...c
 			g.Expect(c.Get(ctx, client.ObjectKeyFromObject(obj), obj)).To(Succeed())
 			g.Expect(obj.GetDeletionTimestamp().IsZero()).ToNot(BeTrue())
 		}
-	}, ReconcilerPropagationTime, RequestInterval).Should(Succeed())
+	}, time.Second).Should(Succeed())
```

Comment:
Why are we reducing the timeout to a second here? It doesn't appear to be a problem, just curious really.

Reply:
I don't think we need a longer timeout -- if it functions with the shorter timeout, we should go with that -- since it allows us to fail-fast